src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <div class="product-card">
+        <h1 class="product-card__content">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the full page, so will be better if you use another tag here
```suggestion
        <h3 class="product-card__content">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h3>
```